docs/src/reference/lib/changelog.rst

```diff
@@ -1,36 +1,44 @@
 Changelog
 =========
 
-21.4.0rc1 (UNRELEASED)
-----------------------
+.. _lib-21.8.0:
+
+21.8.0 (UNRELEASED)
+-------------------
+
+.. start-21.8.0
 
 Added
 *****
 
+* Add ``KlioReadFromPubSub`` and ``KlioWriteToPubSub`` IO transforms.
 * Add default metrics to be collected in Klio's IO transforms.
 * Add default metrics to be collected in Klio's helper transforms.
 * Add default metrics to be collected in Klio's decorators.
-* Add ``KlioReadFromPubSub`` and ``KlioWriteToPubSub`` IO transforms.
-
-Fixed
-*****
+* Add support for using Beam's metrics API directly.
+* Add DirectGKERunner, runs direct runner on GKE with added logic to only ack pub/sub message when:
 
-* ``KlioTriggerUpstream`` no longer raises a pickling error when trying to log.
+    * the pipeline successfully ran, but before any write to event output happens if any,
```

Comment:
If you look at the [rendering of this bit](https://klio--207.org.readthedocs.build/en/207/release_notes/21.8.0.html#id5), it looks like it's quoted. I think you need to remove the new line between lines 19 and 21.